targets/TARGET_Freescale/TARGET_KSDK2_MCUS/TARGET_MCU_K64F/device/TOOLCHAIN_GCC_ARM/MK64FN1M0xxx12.ld

```diff
@@ -52,9 +52,11 @@ ENTRY(Reset_Handler)
 
 __ram_vector_table__ = 1;
 
-/* Heap 1/4 of ram and stack 1/8 */
-__stack_size__ = 0x8000;
-__heap_size__ = 0x10000;
+/* Heap 1/4 of ram and stack 1/8. 0x1000 is reduced from both heap
+ * and stack to accomodate static data introduced by GCOV.
+ */
+__stack_size__ = 0x7000;
+__heap_size__ = 0x09000;
```

Comment:
@c1728p9 I know you and @mazimkhan worked together on this, is there any concern on this affecting existing applications? Or should these values need to be conditionally modified if coverage is enabled?